index.js

```diff
@@ -4,7 +4,50 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    const operator = process.argv[2];
+    const inputs = process.argv.slice(3);
+    let number = Number(inputs[0]);
+
+    if(!operator) {
+        return 'No operation provided...'
+
+    } else if (!inputs.length) {
+        return 'No numbers provided...'
+    }
+    
+    if(operator === 'plus') {
+        for(let i = 1; i < inputs.length; i++) {
+            
+            number += Number(inputs[i]);
+        }
+        return number;
+
+
+    } else if (operator === 'minus') {
+        for(let i = 1; i < inputs.length; i++) {
+            
+           number -= Number(inputs[i]);
+
+        }
+        return number;
+
+    } else if (operator === 'divide') {
```

Comment:
instead of including an if statement for all possible other operators, you could have used your operator variable and interpolation! `Invalid operation: ${operator}`